backend/src/main/resources/application-create-sample-data.yaml

```diff
@@ -181,6 +181,15 @@ simple-report-initialization:
       facility-name: Uptown Clinic
     - link: downTwn
       facility-name: Downtown Clinic
+  patients:
+    - first-name: "SRPatient1"
```

Comment:
nit: any chance that we could use semi-real names like we do with the user profiles?

Reply:
hahah I actually made it intentionally fake sounding so we could know easily the users created on startup but I am happy to make them more real names! 😄

Reply:
Yeah since this data would show up on demo and training having semi believable names might be good, at least in the same manner as the API user fixtures